docs/guides/dws-on-csm/readme.md

```diff
@@ -0,0 +1,185 @@
+---
+authors: Dean Roehrich <dean.roehrich@@hpe.com>
+categories: install
+---
+
+# DWS On CSM: Installation and use of DWS on a CSM cluster
+
+## Install DWS
+
+### DWS Dependencies
+
+Install the DWS dependencies and set the expected node labels prior to installing DWS.
+
+<details>
+<summary>DWS Dependencies</summary>
+    DWS requires kube-rbac-proxy to be present in the cluster's container registry.
+
+```bash
+ncn-m001:~ # podman run --rm --network host quay.io/skopeo/stable:v1.4.1 copy --dest-tls-verify=false docker://gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0 docker://registry.local/kube-rbac-proxy:v0.13.0
+```
```

Comment:
Prefix'ing `ncn-m001:~ #` makes sense from a readability perspective, since you want to communicate that you are running these commands on a CSM node. However, this makes copy/pasting each of these commands more inconvenient than it should be. I'd like to just use the mkdocs button to copy each command, but I have to strip off the prefix every single time.

Reply:
Consider removing the prefix.

Reply:
I also found an interesting discussion about using a tool called Superfences to solve this issue.
Here's the discussion: https://github.com/squidfunk/mkdocs-material/discussions/3427
and here's an example repo with an implementation: https://github.com/flynneva/mkdocs-custom-superfence-example